lib/broccoli/ember-app.js

```diff
@@ -1213,10 +1354,12 @@ EmberApp.prototype._prunedBabelOptions = function() {
   @return {Tree} Merged tree
 */
 EmberApp.prototype.javascript = function() {
-  var deprecate           = this.project.ui.writeDeprecateLine.bind(this.project.ui);
-  var applicationJs       = this.appAndDependencies();
-  var appOutputPath       = this.options.outputPaths.app.js;
-  var appJs               = applicationJs;
+  var deprecate             = this.project.ui.writeDeprecateLine.bind(this.project.ui);
```

Comment:
Separate from Krati's PR: can we all agree that this is a bad style and stop doing it? ❤️